src/gui/controller/CreateCaseController.java

```diff
@@ -0,0 +1,46 @@
+package gui.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TabPane;
+
+public class CreateCaseController extends Controller {
+
+  /**
+   * Tabpane holding the different stages in case-creation
+   */
+  @FXML
+  private TabPane caseTabPane;
```

Comment:
Could be named `tabPane` instead of `caseTabPane`, because we are in the `CreateCase` context,